RJL.HW4.OOP.Classes.Task2/Program.cs

```diff
@@ -10,11 +10,20 @@ class Program
     {
         static void Main(string[] args)
         {
-            Agregat Agregat1 = new Agregat(20);
-            Console.WriteLine($"Agregat contains {Agregat1.AgregatCountDetails} details");
-            int countJunWorker = 5;
-            int countMiddleWorker = 3;
-            int countSeniourWorker = 2;
+            Agregat Agregat1 = new Agregat(20,0);
+            Console.WriteLine($"Agregat contains {Agregat1.GeneralCountDetails} details");
+            Console.WriteLine("-----------------------------------------------------");
+            Worker[] workers1 = GetWorkers(5, 4, 3);
+            Console.WriteLine("Team consists of:");
+            PrintWorkersTeam(workers1);
+            Console.WriteLine("-----------------------------------------------------");
+            Console.WriteLine("Assembly of agregat has been started");
+            Console.WriteLine("-----------------------------------------------------");
+            GetAssemledAgregat(Agregat1, workers1);
```

Comment:
Method which name started with `Get....(...)` should return something. Call `DoAggregatAssembling(..)` or `AssembleAggregat(...)`